src/interface.jl

```diff
@@ -437,46 +425,142 @@ function inv! end
+This means it is either the [`Identity`](@ref)`{O}` with the respect to the corresponding
+[`AbstractGroupOperation`](@ref) `O`, or (approximately) the correct point representation.
 
-The computation can be done in-place of `Z`.
+# See also
+
+[`identity_element`](@ref), [`identity_element!`](@ref)
 """
-function lie_bracket end
-@doc "$(_doc_lie_bracket)"
-function lie_bracket(𝔤::LieAlgebra, X, Y)
-    Z = ManifoldsBase.allocate_result(𝔤, lie_bracket, X, Y)
-    return lie_bracket!(𝔤, Z, X, Y)
+is_identity(G::LieGroup, q)
+
+function is_identity(G::LieGroup{𝔽,O}, h; kwargs...) where {𝔽,O<:AbstractGroupOperation}
+    return ManifoldsBase.isapprox(G, Identity{O}(), h; kwargs...)
+end
+function is_identity(
+    ::LieGroup{𝔽,O}, ::Identity{O}; kwargs...
+) where {𝔽,O<:AbstractGroupOperation}
+    return true
+end
+# any other identity than the fitting one
+function is_identity(
+    G::LieGroup{𝔽,<:AbstractGroupOperation},
+    h::Identity{<:AbstractGroupOperation};
+    kwargs...,
+) where {𝔽}
+    return false
 end
 
-function lie_bracket! end
-@doc "$(_doc_lie_bracket)"
-lie_bracket!(𝔤::LieAlgebra, Z, X, Y)
+"""
+    is_point(G::LieGroup, g; kwargs...)
+
+Check whether `g` is a valid point on the Lie Group `G`.
+This falls back to checking whether `g` is a valid point on `G.manifold`,
+unless `g` is an [`Identity`](@ref). Then, it is checked whether it is the
+idenity element corresponding to `G`.
+"""
+ManifoldsBase.is_point(G::LieGroup, g; kwargs...)
+
+_doc_is_vector = """
+    is_vector(G::LieGroup, X; kwargs...)
+    is_vector(G::LieGroup{𝔽,O}, e::Indentity{O}, X; kwargs...)
+
+Check whether `X` is a tangent vector, that is an element of the |`LieAlgebra`](@ref)
+of `G`.
+The first variant calls [`is_point`](@extref ManifoldsBase.is_point) on the [`LieAlgebra`](@ref) `𝔤` of `G`.
+The second variant calls [`is_vector`](@extref ManifoldsBase.is_vector) on the $(_link(:AbstractManifold)) at the [`identity_element`](@ref).
+
+All kwyword arguments are passed on to the corresponding call
+"""
+
+@doc "$(_doc_is_vector)"
+ManifoldsBase.is_vector(G::LieGroup, X; kwargs...) =
+    ManifoldsBase.is_point(LieAlgebra(G), X; kwargs...)
+
+@doc "$(_doc_is_vector)"
+function ManifoldsBase.is_vector(G::LieGroup{𝔽,O}, e::Identity{O}, X; kwargs...) where {𝔽,O}
+    return ManifoldsBase.is_vector(G.manifold, identity_element(G), X; kwargs...)
+end
+
+"""
+    isapprox(M::LieGroup, g, h; kwargs...)
+
+Check if points `g` and `h` from [`LieGroup`](@ref) are approximately equal.
+this function calls the corresponding $(_link(:isapprox)) on the $(_link(:AbstractManifold))
+after handling the cases where one or more
+of the points are the [`Identity`](@ref).
+All keyword argments are passed to this function as well.
+"""
+ManifoldsBase.isapprox(G::LieGroup, g, h; kwargs...) = isapprox(G.manifold, g, h; kwargs...)
+function ManifoldsBase.isapprox(
+    G::LieGroup{𝔽,O}, g::Identity{<:O}, h; kwargs...
```

Comment:
```suggestion
    G::LieGroup{𝔽,O}, g::Identity{O}, h; kwargs...
```

Reply:
For these I also never know when to put those and when not. The last 2 I added were when Aqua complained.